.github/workflows/ci.yml

```
@@ -1,619 +1,73 @@
+          fi
+
+      - name: Upload coverage artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: coverage-report
+          path: build/filtered.info
+
+      - name: PR Coverage Comment
+        if: github.event_name == 'pull_request'
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected key

(syntax-check)